packages/devextreme/js/__internal/grids/grid_core/data_controller/m_data_controller.ts

```diff
@@ -1640,7 +1638,7 @@ export class DataController extends DataHelperMixin(modules.Controller) {
     if (options === true) {
       options = { reload: true, changesOnly: true };
     } else if (!options) {
-      options = { lookup: true, selection: true, reload: true };
+      options = { reload: true, lookup: true };
```

Comment:
I'm afraid in this case we can get extra requests. For example, when we have remote data and initially set `selectedRowKeys` with data keys that are not loaded yet, then when calling the `dataGridInstance.refresh(true)` method there will be an extra request for the selected data.